File: PY/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    __jenis = ""
    __bahan = ""
    __warna = ""

    def __init__(self, jenis = "", bahan = "", warna = ""):
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna
                    
    def getJenis(self):
        return self.__jenis
    
    def setJenis(self, jenis):
        self.__jenis = jenis
    
    def getBahan(self):
        return self.__bahan
    
    def setBahan(self, bahan):
        self.__bahan = bahan
    
    def getWarna(self):
        return self.__warna
    
    def setWarna(self, warna):
        self.__warna = warna
    
    def getlJenis(self):
        return len(self.__jenis)
    def getlBahan(self):
        return len(self.__bahan)
    def getlWarna(self):
        return len(self.__warna)

File: PHP/Aksesoris.php
<?php
    require_once 'Petshop.php';
    class Aksesoris extends Petshop
    {
        private $jenis;
        private $bahan;
        private $warna;
        public function __construct($jenis="", $bahan="", $warna=0){
            $this->jenis = $jenis;
            $this->bahan = $bahan;
            $this->warna = $warna;
        }

        public function setJenis($jenis){
            $this->jenis = $jenis;
        }
        public function getJenis(){
            return $this->jenis;
        }
        public function setBahan($bahan){
            $this->bahan = $bahan;
        }
        public function getBahan(){
            return $this->bahan;
        }
        public function setWarna($warna){
            $this->warna = $warna;
        }
        public function getWarna(){
            return $this->warna;
        }
    }
?>